.github/ISSUE_TEMPLATE/feature_request.md

```
@@ -0,0 +1,59 @@
+---
+type: "feature"
+priority: ""              # critical | high | medium | low
+size: ""                  # tiny (<1 file) | small (1-3 files) | medium (3-8 files)
+platform: []              # ios | android | web | desktop | sdk | server | docs
+files:
+  read: []                # Files for context
+  write: []               # Files to create or modify
+verify: []                # Commands to confirm completion
+---
+
+# [Add/Implement] [WHAT] [WHERE]
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)